contracts/src/core/AlignedLayerServiceManager.sol

```diff
@@ -124,6 +128,8 @@ contract AlignedLayerServiceManager is ServiceManagerBase, BLSSignatureChecker {
             "Signatories do not own at least threshold percentage of a quorum"
         );
 
+        batchesState[batchMerkleRoot].responded = true;
```

Comment:
This line should be right after the `require` statement it is involved in (line 104), to follow CEI pattern and thus avoid possible re-entrancy attacks.
Even though this function has not much value to be exploited with a reentrancy, it is always a good idea to follow precaution and safety patterns 👍